project-management-infrastructure/src/main/java/life/qbic/projectmanagement/infrastructure/experiment/QbicExperimentRepo.java

```diff
@@ -12,4 +14,7 @@ public interface QbicExperimentRepo extends JpaRepository<Experiment, Experiment
 
   List<Experiment> findExperimentByExperimentId(ExperimentId id);
 
+  @Query(value = "SELECT project FROM experiments_datamanager WHERE id = ?1",
+      nativeQuery = true)
```

Comment:
because due to the current table structure this is not as straightforward as the other queries and we didn't want to make big changes to the experiment or project table